pom.xml

```diff
@@ -5,7 +5,7 @@
 	<groupId>Mambu-APIs-Java</groupId>
 	<artifactId>Mambu-APIs-Java</artifactId>
 	<name>Mambu Java APIs SDK</name>
-	<version>9.4-bin</version>
+	<version>9.5-bin</version>
```

Comment:
Should we delete 9.2? Also, in lib folder we have models starting from 8.0. Do we want to keep them all?

Reply:
deleted them, thanks!